datasource.js

```diff
@@ -340,7 +340,13 @@ function (angular, _, sdk, dateMath, kbn) {
     if (target.tags) {
       query.tags = angular.copy(target.tags);
       _.forOwn(query.tags, function(value, key) {
-        query.tags[key] = _.map(value, function(tag) { return self.templateSrv.replace(tag); });
+        query.tags[key] = _.flatten(_.map(value, function(tag) {
+          var replacement = self.templateSrv.replace(tag);
+          if (replacement.indexOf('{') == 0 && replacement.lastIndexOf('}') == replacement.length - 1) {
```

Comment:
When the templateSrv.replacement is called, the value comes back as a string that looks like "{value1,value2}".  Kairos accepts an array of tags.  I believe that the original code wrapped single values in an array and thus needed the "flatten" call.  This should not break existing functionality.